src/PrestaShopBundle/Resources/views/Admin/Module/Includes/action_menu.html.twig

```diff
@@ -41,34 +41,49 @@
+          {% if
+            (level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_UPDATE')) or
+            (level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE') and (url_active not in ['enable', 'disable', 'configure', 'install', 'upgrade']))
+          %}
+            disabled="disabled"
+          {% endif %}>
             {{ url_active|capitalize|replace({'_': " "})|trans({}, 'Admin.Actions') }}
         </button>
         {% if urls|length > 1 %}
         <input type="hidden" class="btn">
         {% endif %}
       </form>
       {% if (urls|length > 1) %}
-        {% if (level > constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_CREATE')) or ((('configure' in urls|keys) and ('upgrade' in urls|keys)) and  url_active in ['configure', 'install', 'upgrade']) %}
-            <button type="button" class="btn btn-primary-outline  dropdown-toggle light-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-              <span class="caret"></span>
-              <span class="sr-only">{{ 'Toggle Dropdown'|trans({}, 'Admin.Modules.Feature') }}</span>
-            </button>
-            <div class="dropdown-menu">
-              {% for module_action, module_url in urls %}
-                {% if module_action != url_active %}
-                  {% if ((level >= constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE')) or ((level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE')) and (module_action in ['configure', 'install', 'upgrade']))) %}
-                    <li>
-                      <form method="post" action="{{ module_url }}">
-                        <button type="submit" class="dropdown-item module_action_menu_{{ module_action }}" data-confirm_modal="module-modal-confirm-{{ name }}-{{ module_action }}">
-                          {{module_action|capitalize|replace({'_': " "})|trans({}, 'Admin.Actions')}}
-                        </button>
-                      </form>
-                    </li>
-                  {% endif %}
+        {% if
+          (level > constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_CREATE')) or
+          (
+            (('configure' in urls|keys) and (('upgrade' in urls|keys) or ('enable' in urls|keys) or ('disable' in urls|keys))) and
+            url_active in ['enable', 'disable', 'configure', 'install', 'upgrade']
+          )
+        %}
+          <button type="button" class="btn btn-primary-outline  dropdown-toggle light-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <span class="caret"></span>
+            <span class="sr-only">{{ 'Toggle Dropdown'|trans({}, 'Admin.Modules.Feature') }}</span>
+          </button>
+          <div class="dropdown-menu">
+            {% for module_action, module_url in urls %}
+              {% if module_action != url_active %}
+                {% if
```

Comment:
It was already like that. I can understand because the two tests are absolutely unrelated each other